### PR TITLE
Layout fixes

### LINKED ISSUE #567: Fix modal layout problems

### PR DESCRIPTION
**Pull request by clemens-msupply**
Fixes various smaller issues:
- updates schema
- layout spacing in the category modal
- more consistent margins
- wider left column in forms
- fix encounter query in the encounter chart
- fix initial sort in useUrlQueryParams hook (@mark-prins please double check)
- React warning for unstable values

Closes #567

### CODE REVIEW

**Comment by mark-prins** [comment]
Just confirming that the base should be `feature/programs` ( 11 files 😁 ) and not `develop` ( 382 files 😱 )

File: client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts [hunk 1]
```diff
@@ -76,8 +76,8 @@ export const useUrlQueryParams = ({
     offset: urlQuery.page ? (urlQuery.page - 1) * rowsPerPage : 0,
     first: rowsPerPage,
     sortBy: {
-      key: urlQuery.sort ?? initialSort,
-      direction: urlQuery.dir ?? 'asc',
+      key: urlQuery.sort ?? initialSort?.key,
```
Comment (mark-prins) [inline client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts:79]: annoying that the UrlQuery hooks are using 'dir' and the sort is using 'direction'. Was hoping we could just use `sortBy: initialSort ?? { ... `

anyhoo.. 
would you mind updating line `39` to 
```
    updateQuery({ sort, dir: dir === 'desc' ? 'desc' : '' });
```

that makes the url query shorter and more consistent.

**Review by mark-prins — APPROVED** [review]
👍  Looks great!

I have an additional request:
can you remove the `queryParamsStore` from the `TableProvider` in `/packages/system/src/Patient/ListView/ListView.tsx`?

not required, I think. and the initial sort being set is incorrect.

I have also updated the initial sort in `usePatients` to be by `lastName` as I don't find code that helpful 🤷

File: client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts (hunk 1 above)
Comment (clemens-msupply) [reply client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts:79]: > I have also updated the initial sort in `usePatients` to be by `lastName` as I don't find code that helpful shrug

Sounds good, did you do it in this PR? no commit here...

Comment (clemens-msupply) [reply client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts:79]: thanks good ideas, I made these changes

Comment (mark-prins) [reply client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts:79]: oh no!
@clemens-msupply I have found a problem.. this has broken our column sort. If you try to change direction on a column, it won't work.

The problem is that when sorting ascending, we remove the `dir` parameter
Then, in this line

```
 direction: urlQuery.dir ?? initialSort?.dir ?? 'asc',
```

the `urlQuery.dir` will return undefined, and so will always return the `initialSort?.dir`. If the initial sort has been set, then you are unable to change direction!

I've put a suggestion on [this PR](https://github.com/openmsupply/open-msupply/pull/2237) and tested the initial sort, and that's working for me. Can you have a quick check as well? afaics the `useEffect` should be handling the initial sort without the need for this change

**Pull request merged**